{"tensorflow.python.ops.math_ops.reduce_min_v1": ["tensorflow.python.util.deprecation.deprecated_argument_lookup", "tensorflow.python.ops.math_ops.reduce_min"], "tensorflow.python.framework.ops.name_scope.__init__": ["<builtin>.TypeError", "<builtin>.isinstance", "<builtin>.type", "tensorflow.python.eager.context.context", "<builtin>.hasattr"], "tensorflow.python.training.saver.Saver.__init__": ["<builtin>.ValueError", "tensorflow.python.eager.context.executing_eagerly", "tensorflow.python.platform.tf_logging.warning", "<builtin>.RuntimeError", "time.time", "tensorflow.python.training.saver.Saver.build", "tensorflow.python.training.saver.Saver._check_saver_def"], "tensorflow.python.framework.ops.Graph.__init__": ["threading.RLock", "tensorflow.python.util.lock_util.GroupLock.__init__", "threading.local", "tensorflow.python.framework.traceable_stack.TraceableStack.__init__", "collections.OrderedDict", "tensorflow.python.util.object_identity.ObjectIdentitySet.__init__", "<builtin>.set", "tensorflow.python.framework.ops.uid", "tensorflow.python.framework.op_def_registry.get_registered_ops", "tensorflow.python.framework.c_api_util.ScopedTFGraph.__init__", "tensorflow.python.tf2.enabled", "tensorflow.python.framework.ops.Graph.switch_to_thread_local"], "tensorflow.python.ops.nn_ops.dropout": ["<builtin>.ValueError", "tensorflow.python.util.deprecation.deprecated_argument_lookup", "tensorflow.python.ops.nn_ops.dropout_v2"], "tensorflow.contrib.layers.python.layers.initializers.xavier_initializer": ["tensorflow.contrib.layers.python.layers.initializers.variance_scaling_initializer"], "tensorflow.python.ops.nn_ops.conv2d": ["tensorflow.python.util.deprecation.deprecated_argument_lookup", "tensorflow.python.ops.nn_ops._convert_padding", "tensorflow.python.ops.nn_ops._get_sequence", "tensorflow.python.ops.gen_nn_ops.conv2d"], "tensorflow.python.ops.random_ops.random_normal": ["tensorflow.python.framework.ops.name_scope.__init__", "tensorflow.python.framework.tensor_util.shape_tensor", "tensorflow.python.framework.ops.convert_to_tensor", "tensorflow.python.framework.random_seed.get_seed", "tensorflow.python.ops.gen_random_ops.random_standard_normal", "tensorflow.python.framework.tensor_util.maybe_set_static_shape"], "tensorflow.python.summary.writer.writer.FileWriter.__init__": ["tensorflow.python.eager.context.executing_eagerly", "<builtin>.RuntimeError", "tensorflow.python.summary.writer.event_file_writer_v2.EventFileWriterV2.__init__", "tensorflow.python.summary.writer.event_file_writer.EventFileWriter.__init__", "tensorflow.python.summary.writer.writer.SummaryToEventTransformer.__init__", "<builtin>.super"], "tensorflow.python.ops.array_ops.reshape": ["tensorflow.python.ops.gen_array_ops.reshape", "tensorflow.python.framework.tensor_util.maybe_set_static_shape"], "tensorflow.python.training.adam.AdamOptimizer.__init__": ["<builtin>.super"], "tensorflow.python.ops.nn_ops.max_pool": ["tensorflow.python.util.deprecation.deprecated_argument_lookup", "tensorflow.python.framework.ops.name_scope.__init__", "tensorflow.python.ops.nn_ops._get_sequence", "tensorflow.python.ops.gen_nn_ops.max_pool", "<builtin>.ValueError", "numpy.isscalar", "<builtin>.isinstance", "<builtin>.any"], "tensorflow.python.ops.math_ops.argmax": ["tensorflow.python.util.deprecation.deprecated_argument_lookup", "tensorflow.python.ops.math_ops.argmax_v2"], "tensorflow.python.ops.math_ops.cast": ["tensorflow.python.framework.dtypes.as_dtype", "<builtin>.isinstance", "tensorflow.python.framework.ops.name_scope.__init__", "tensorflow.python.ops.math_ops.cast", "tensorflow.python.framework.sparse_tensor.SparseTensor.__init__", "tensorflow.python.platform.tf_logging.warn", "tensorflow.python.framework.ops.convert_to_tensor", "tensorflow.python.ops.gen_math_ops.cast"], "tensorflow.python.ops.nn_impl.sigmoid_cross_entropy_with_logits": ["tensorflow.python.ops.nn_ops._ensure_xent_args", "tensorflow.python.framework.ops.name_scope.__init__", "tensorflow.python.framework.ops.convert_to_tensor", "tensorflow.python.ops.array_ops.zeros_like", "tensorflow.python.ops.array_ops.where", "<builtin>.ValueError"], "tensorflow.python.ops.gen_math_ops.add": ["tensorflow.python.eager.context.context", "six.raise_from", "tensorflow.python.ops.gen_math_ops.add_eager_fallback", "tensorflow.python.eager.core._status_to_exception", "tensorflow.python.util.dispatch.dispatch", "tensorflow.python.eager.execute.record_gradient"], "tensorflow.python.ops.math_ops.equal": ["tensorflow.python.ops.gen_math_ops.equal"], "tensorflow.python.ops.nn_ops.bias_add": ["tensorflow.python.framework.ops.name_scope.__init__", "tensorflow.python.ops.gen_nn_ops.bias_add", "tensorflow.python.eager.context.executing_eagerly", "tensorflow.python.framework.ops.convert_to_tensor", "<builtin>.ValueError"], "tensorflow.python.ops.math_ops.reduce_mean_v1": ["tensorflow.python.util.deprecation.deprecated_argument_lookup", "tensorflow.python.ops.math_ops.reduce_mean"], "tensorflow.python.ops.array_ops.placeholder": ["tensorflow.python.eager.context.executing_eagerly", "<builtin>.RuntimeError", "tensorflow.python.ops.gen_array_ops.placeholder"], "tensorflow.python.ops.variables.global_variables_initializer": ["tensorflow.python.eager.context.executing_eagerly", "tensorflow.python.ops.variables.variables_initializer", "tensorflow.python.ops.variables.global_variables"], "tensorflow.python.client.session.Session.__init__": ["tensorflow.python.client.session.BaseSession.__init__", "<builtin>.super"], "tensorflow.python.ops.gen_nn_ops.relu": ["tensorflow.python.eager.context.context", "six.raise_from", "tensorflow.python.ops.gen_nn_ops.relu_eager_fallback", "tensorflow.python.eager.core._status_to_exception", "tensorflow.python.util.dispatch.dispatch", "tensorflow.python.eager.execute.record_gradient"], "tensorflow.python.ops.math_ops.matmul": ["tensorflow.python.framework.ops.name_scope.__init__", "tensorflow.python.eager.context.executing_eagerly", "tensorflow.python.compat.compat.forward_compatible", "<builtin>.ValueError", "tensorflow.python.framework.ops.convert_to_tensor", "tensorflow.python.ops.math_ops.conj", "tensorflow.python.ops.gen_math_ops.mat_mul", "<builtin>.isinstance", "tensorflow.python.ops.math_ops.cast", "<builtin>.len"], "tensorflow.python.ops.variable_scope.get_variable": ["tensorflow.python.ops.variable_scope._get_default_variable_store", "tensorflow.python.ops.variable_scope.get_variable_scope"], "tensorflow.python.util.deprecation.deprecated_argument_lookup": ["<builtin>.ValueError"], "tensorflow.python.ops.math_ops.reduce_min": ["tensorflow.python.ops.math_ops._may_reduce_to_scalar", "tensorflow.python.ops.gen_math_ops._min", "tensorflow.python.ops.math_ops._ReductionDims"], "tensorflow.python.eager.context.context": ["tensorflow.python.eager.context._create_context"], "tensorflow.python.eager.context.executing_eagerly": ["tensorflow.python.eager.context.context_safe", "tensorflow.python.eager.context.context"], "tensorflow.python.platform.tf_logging.warning": ["tensorflow.python.platform.tf_logging.get_logger"], "tensorflow.python.training.saver.Saver.build": ["tensorflow.python.eager.context.executing_eagerly", "<builtin>.RuntimeError", "tensorflow.python.training.saver.Saver._build"], "tensorflow.python.training.saver.Saver._check_saver_def": ["<builtin>.isinstance", "<builtin>.ValueError", "tensorflow.python.eager.context.executing_eagerly", "<builtin>.str"], "tensorflow.python.util.lock_util.GroupLock.__init__": ["<builtin>.ValueError", "threading.Condition", "threading.Lock"], "tensorflow.python.util.object_identity.ObjectIdentitySet.__init__": ["<builtin>.set", "tensorflow.python.util.object_identity.ObjectIdentitySet._wrap_key", "<builtin>.list"], "tensorflow.python.tf2.enabled": ["os.getenv"], "tensorflow.python.ops.nn_ops.dropout_v2": ["tensorflow.python.framework.ops.name_scope.__init__", "tensorflow.python.framework.ops.convert_to_tensor", "<builtin>.isinstance", "tensorflow.python.eager.context.executing_eagerly", "tensorflow.python.ops.nn_ops._get_noise_shape", "tensorflow.python.ops.random_ops.random_uniform", "<builtin>.ValueError", "tensorflow.python.ops.math_ops.cast", "tensorflow.python.platform.tf_logging.log_first_n", "tensorflow.python.framework.tensor_util.constant_value"], "tensorflow.contrib.layers.python.layers.initializers.variance_scaling_initializer": ["<builtin>.TypeError", "<builtin>.float", "math.sqrt", "tensorflow.python.ops.random_ops.random_uniform", "tensorflow.python.ops.random_ops.truncated_normal", "<builtin>.len"], "tensorflow.python.ops.nn_ops._convert_padding": ["<builtin>.ValueError", "<builtin>.isinstance", "<builtin>.enumerate", "<builtin>.len"], "tensorflow.python.ops.nn_ops._get_sequence": ["<builtin>.isinstance", "<builtin>.len", "<builtin>.list", "<builtin>.ValueError"], "tensorflow.python.ops.gen_nn_ops.conv2d": ["tensorflow.python.eager.context.context", "six.raise_from", "tensorflow.python.ops.gen_nn_ops.conv2d_eager_fallback", "tensorflow.python.eager.core._status_to_exception", "<builtin>.isinstance", "<builtin>.TypeError", "tensorflow.python.eager.execute.make_int", "tensorflow.python.eager.execute.make_str", "tensorflow.python.eager.execute.make_bool", "tensorflow.python.eager.execute.record_gradient"], "tensorflow.python.framework.tensor_util.shape_tensor": ["<builtin>.isinstance", "<builtin>.tuple", "<builtin>.map", "tensorflow.python.framework.ops.convert_to_tensor"], "tensorflow.python.framework.ops.convert_to_tensor": ["tensorflow.python.util.deprecation.deprecated_argument_lookup", "tensorflow.python.framework.ops.convert_to_tensor_v2"], "tensorflow.python.framework.random_seed.get_seed": ["tensorflow.python.eager.context.executing_eagerly", "tensorflow.python.eager.context.global_seed", "tensorflow.python.framework.ops.get_default_graph", "<builtin>.hasattr", "tensorflow.python.framework.random_seed._truncate_seed", "tensorflow.python.eager.context.internal_operation_seed"], "tensorflow.python.ops.gen_random_ops.random_standard_normal": ["tensorflow.python.eager.context.context", "six.raise_from", "tensorflow.python.ops.gen_random_ops.random_standard_normal_eager_fallback", "tensorflow.python.eager.core._status_to_exception", "tensorflow.python.eager.execute.make_type", "tensorflow.python.eager.execute.make_int", "tensorflow.python.eager.execute.record_gradient"], "tensorflow.python.framework.tensor_util.maybe_set_static_shape": ["tensorflow.python.framework.tensor_util.is_tensor", "tensorflow.python.framework.tensor_util.shape_tensor", "tensorflow.python.framework.tensor_util.constant_value_as_shape", "tensorflow.python.eager.context.executing_eagerly", "tensorflow.python.framework.ops.get_default_graph"], "tensorflow.python.summary.writer.event_file_writer_v2.EventFileWriterV2.__init__": ["tensorflow.python.framework.ops.name_scope.__init__", "tensorflow.python.ops.summary_ops_v2.create_file_writer", "tensorflow.python.ops.summary_ops_v2.always_record_summaries", "tensorflow.python.ops.array_ops.placeholder_with_default", "tensorflow.python.ops.summary_ops_v2.import_event", "tensorflow.python.framework.constant_op.constant"], "tensorflow.python.summary.writer.event_file_writer.EventFileWriter.__init__": ["<builtin>.str", "six.moves.queue.Queue", "tensorflow.python.util.compat.as_bytes", "os.path.join", "tensorflow.python.summary.writer.event_file_writer.EventFileWriter._get_sentinel_event", "tensorflow.python.summary.writer.event_file_writer._EventLoggerThread.__init__"], "tensorflow.python.summary.writer.writer.SummaryToEventTransformer.__init__": ["tensorflow.python.summary.writer.writer.SummaryToEventTransformer.add_graph", "tensorflow.python.summary.writer.writer.SummaryToEventTransformer.add_meta_graph", "<builtin>.isinstance", "tensorflow.python.framework.meta_graph.create_meta_graph_def", "<builtin>.set"], "tensorflow.python.ops.gen_array_ops.reshape": ["tensorflow.python.eager.context.context", "six.raise_from", "tensorflow.python.ops.gen_array_ops.reshape_eager_fallback", "tensorflow.python.eager.core._status_to_exception", "tensorflow.python.eager.execute.record_gradient"], "tensorflow.python.ops.gen_nn_ops.max_pool": ["tensorflow.python.eager.context.context", "six.raise_from", "tensorflow.python.ops.gen_nn_ops.max_pool_eager_fallback", "tensorflow.python.eager.core._status_to_exception", "<builtin>.isinstance", "<builtin>.TypeError", "tensorflow.python.eager.execute.make_int", "tensorflow.python.eager.execute.make_str", "tensorflow.python.eager.execute.record_gradient"], "tensorflow.python.ops.math_ops.argmax_v2": ["tensorflow.python.ops.gen_math_ops.arg_max"], "tensorflow.python.framework.dtypes.as_dtype": ["<builtin>.isinstance", "<builtin>.TypeError"], "tensorflow.python.framework.sparse_tensor.SparseTensor.__init__": ["tensorflow.python.framework.ops.name_scope.__init__", "tensorflow.python.framework.ops.convert_to_tensor", "tensorflow.python.framework.ops.internal_convert_to_tensor"], "tensorflow.python.platform.tf_logging.warn": ["tensorflow.python.platform.tf_logging.get_logger"], "tensorflow.python.ops.gen_math_ops.cast": ["tensorflow.python.eager.context.context", "six.raise_from", "tensorflow.python.ops.gen_math_ops.cast_eager_fallback", "tensorflow.python.eager.core._status_to_exception", "tensorflow.python.eager.execute.make_type", "tensorflow.python.eager.execute.make_bool", "tensorflow.python.eager.execute.record_gradient"], "tensorflow.python.ops.nn_ops._ensure_xent_args": ["<builtin>.ValueError"], "tensorflow.python.ops.array_ops.zeros_like": ["tensorflow.python.ops.array_ops.zeros_like_impl"], "tensorflow.python.ops.array_ops.where": ["tensorflow.python.framework.ops.name_scope.__init__", "tensorflow.python.framework.ops.convert_to_tensor", "tensorflow.python.ops.gen_array_ops.where", "tensorflow.python.ops.gen_math_ops.select", "<builtin>.ValueError"], "tensorflow.python.ops.gen_math_ops.add_eager_fallback": ["tensorflow.python.eager.context.context", "tensorflow.python.eager.execute.args_to_matching_eager", "tensorflow.python.eager.execute.record_gradient"], "tensorflow.python.util.dispatch.dispatch": ["<builtin>.getattr"], "tensorflow.python.ops.gen_math_ops.equal": ["tensorflow.python.eager.context.context", "six.raise_from", "tensorflow.python.ops.gen_math_ops.equal_eager_fallback", "tensorflow.python.eager.core._status_to_exception", "tensorflow.python.eager.execute.make_bool", "tensorflow.python.eager.execute.record_gradient"], "tensorflow.python.ops.gen_nn_ops.bias_add": ["tensorflow.python.eager.context.context", "six.raise_from", "tensorflow.python.ops.gen_nn_ops.bias_add_eager_fallback", "tensorflow.python.eager.core._status_to_exception", "tensorflow.python.eager.execute.make_str", "tensorflow.python.eager.execute.record_gradient"], "tensorflow.python.ops.math_ops.reduce_mean": ["tensorflow.python.ops.math_ops._may_reduce_to_scalar", "tensorflow.python.ops.gen_math_ops.mean", "tensorflow.python.ops.math_ops._ReductionDims"], "tensorflow.python.ops.gen_array_ops.placeholder": ["tensorflow.python.eager.context.context", "six.raise_from", "tensorflow.python.ops.gen_array_ops.placeholder_eager_fallback", "tensorflow.python.eager.core._status_to_exception", "tensorflow.python.eager.execute.make_type", "tensorflow.python.eager.execute.make_shape", "tensorflow.python.eager.execute.record_gradient"], "tensorflow.python.ops.variables.variables_initializer": ["tensorflow.python.ops.control_flow_ops.group", "tensorflow.python.eager.context.executing_eagerly"], "tensorflow.python.ops.variables.global_variables": ["tensorflow.python.framework.ops.get_collection"], "tensorflow.python.client.session.BaseSession.__init__": ["tensorflow.python.eager.monitoring.Counter.get_cell", "tensorflow.python.framework.ops.get_default_graph", "<builtin>.isinstance", "<builtin>.TypeError", "<builtin>.type", "tensorflow.python.util.compat.as_bytes", "threading.Lock", "tensorflow.python.eager.context.context"], "tensorflow.python.ops.gen_nn_ops.relu_eager_fallback": ["tensorflow.python.eager.context.context", "tensorflow.python.eager.execute.args_to_matching_eager", "tensorflow.python.eager.execute.record_gradient"], "tensorflow.python.compat.compat.forward_compatible": ["tensorflow.python.compat.compat._date_to_date_number"], "tensorflow.python.ops.math_ops.conj": ["<builtin>.isinstance", "tensorflow.python.framework.ops.name_scope.__init__", "tensorflow.python.framework.ops.convert_to_tensor", "tensorflow.python.ops.gen_math_ops.conj", "<builtin>.TypeError"], "tensorflow.python.ops.gen_math_ops.mat_mul": ["tensorflow.python.eager.context.context", "six.raise_from", "tensorflow.python.ops.gen_math_ops.mat_mul_eager_fallback", "tensorflow.python.eager.core._status_to_exception", "tensorflow.python.eager.execute.make_bool", "tensorflow.python.eager.execute.record_gradient"], "tensorflow.python.ops.variable_scope._get_default_variable_store": ["tensorflow.python.framework.ops.get_collection", "tensorflow.python.ops.variable_scope._VariableStore.__init__", "tensorflow.python.framework.ops.add_to_collection"], "tensorflow.python.ops.variable_scope.get_variable_scope": ["tensorflow.python.ops.variable_scope.get_variable_scope_store"], "tensorflow.python.ops.math_ops._may_reduce_to_scalar": ["tensorflow.python.framework.common_shapes.has_fully_defined_shape"], "tensorflow.python.ops.gen_math_ops._min": ["tensorflow.python.eager.context.context", "six.raise_from", "tensorflow.python.ops.gen_math_ops._min_eager_fallback", "tensorflow.python.eager.core._status_to_exception", "tensorflow.python.eager.execute.make_bool", "tensorflow.python.eager.execute.record_gradient"], "tensorflow.python.ops.math_ops._ReductionDims": ["<builtin>.ValueError", "tensorflow.python.framework.common_shapes.rank", "tensorflow.python.ops.math_ops.range", "tensorflow.python.framework.constant_op.constant", "<builtin>.isinstance", "tensorflow.python.ops.array_ops.rank", "numpy.arange"], "tensorflow.python.eager.context._create_context": ["tensorflow.python.eager.context.Context.__init__", "tensorflow.python.eager.context._set_context_locked"], "tensorflow.python.platform.tf_logging.get_logger": ["logging.getLogger", "logging.StreamHandler", "logging.Formatter"], "tensorflow.python.training.saver.Saver._build": ["tensorflow.python.eager.context.executing_eagerly", "tensorflow.python.training.saver.BaseSaverBuilder._build_internal", "tensorflow.python.training.saver.BulkSaverBuilder.__default_init__", "tensorflow.python.ops.variables._all_saveable_objects", "tensorflow.python.framework.ops.prepend_name_scope", "<builtin>.ValueError", "tensorflow.python.training.saver.Saver._check_saver_def", "time.time"], "tensorflow.python.util.object_identity.ObjectIdentitySet._wrap_key": ["tensorflow.python.util.object_identity._ObjectIdentityWrapper.__init__"], "tensorflow.python.ops.nn_ops._get_noise_shape": ["tensorflow.python.ops.array_ops.shape", "tensorflow.python.framework.tensor_shape.as_shape", "<builtin>.enumerate", "tensorflow.python.framework.tensor_shape.TensorShape.__init__", "<builtin>.len"], "tensorflow.python.ops.random_ops.random_uniform": ["tensorflow.python.framework.dtypes.as_dtype", "<builtin>.ValueError", "tensorflow.python.framework.ops.name_scope.__init__", "tensorflow.python.framework.tensor_util.shape_tensor", "tensorflow.python.framework.ops.convert_to_tensor", "tensorflow.python.framework.random_seed.get_seed", "tensorflow.python.framework.tensor_util.maybe_set_static_shape", "tensorflow.python.ops.gen_random_ops.random_uniform_int", "tensorflow.python.ops.gen_random_ops.random_uniform"], "tensorflow.python.platform.tf_logging.log_first_n": ["tensorflow.python.platform.tf_logging._GetNextLogCountPerToken", "tensorflow.python.platform.tf_logging._GetFileAndLine", "tensorflow.python.platform.tf_logging.log_if"], "tensorflow.python.framework.tensor_util.constant_value": ["<builtin>.isinstance", "tensorflow.python.framework.tensor_util.is_tensor", "tensorflow.python.framework.tensor_util._ConstantValue"], "tensorflow.python.ops.random_ops.truncated_normal": ["tensorflow.python.framework.ops.name_scope.__init__", "tensorflow.python.framework.tensor_util.shape_tensor", "tensorflow.python.framework.ops.convert_to_tensor", "tensorflow.python.framework.random_seed.get_seed", "tensorflow.python.ops.gen_random_ops.truncated_normal", "tensorflow.python.framework.tensor_util.maybe_set_static_shape"], "tensorflow.python.ops.gen_nn_ops.conv2d_eager_fallback": ["tensorflow.python.eager.context.context", "<builtin>.isinstance", "<builtin>.TypeError", "tensorflow.python.eager.execute.make_int", "tensorflow.python.eager.execute.make_str", "tensorflow.python.eager.execute.make_bool", "tensorflow.python.eager.execute.args_to_matching_eager", "tensorflow.python.eager.execute.record_gradient"], "tensorflow.python.eager.execute.make_int": ["<builtin>.isinstance", "<builtin>.TypeError", "<builtin>.repr", "<builtin>.int"], "tensorflow.python.eager.execute.make_str": ["<builtin>.isinstance", "<builtin>.TypeError", "<builtin>.repr", "tensorflow.python.util.compat.as_bytes"], "tensorflow.python.eager.execute.make_bool": ["<builtin>.isinstance", "<builtin>.TypeError", "<builtin>.repr"], "tensorflow.python.framework.ops.convert_to_tensor_v2": ["tensorflow.python.framework.ops.internal_convert_to_tensor"], "tensorflow.python.eager.context.global_seed": ["tensorflow.python.eager.context.context"], "tensorflow.python.framework.ops.get_default_graph": ["tensorflow.python.framework.ops._DefaultGraphStack.get_default"], "tensorflow.python.eager.context.internal_operation_seed": ["tensorflow.python.eager.context.context"], "tensorflow.python.ops.gen_random_ops.random_standard_normal_eager_fallback": ["tensorflow.python.eager.context.context", "tensorflow.python.eager.execute.make_type", "tensorflow.python.eager.execute.make_int", "tensorflow.python.eager.execute.args_to_matching_eager", "tensorflow.python.eager.execute.record_gradient"], "tensorflow.python.eager.execute.make_type": ["tensorflow.python.framework.dtypes.as_dtype", "<builtin>.TypeError", "<builtin>.repr"], "tensorflow.python.framework.tensor_util.is_tensor": ["<builtin>.isinstance", "tensorflow.python.framework.ops.is_dense_tensor_like", "<builtin>.getattr"], "tensorflow.python.framework.tensor_util.constant_value_as_shape": ["<builtin>.isinstance", "tensorflow.python.framework.tensor_shape.as_shape", "tensorflow.python.framework.tensor_util.constant_value", "tensorflow.python.framework.tensor_shape.unknown_shape", "<builtin>.ValueError", "tensorflow.python.framework.tensor_shape.TensorShape.__init__", "tensorflow.python.framework.tensor_shape.TensorShape.concatenate", "tensorflow.python.framework.tensor_shape.Dimension.__init__", "tensorflow.python.framework.tensor_util.constant_value_as_shape", "<builtin>.hasattr", "<builtin>.enumerate", "tensorflow.python.framework.tensor_shape.TensorShape.merge_with"], "tensorflow.python.ops.summary_ops_v2.create_file_writer": ["tensorflow.python.ops.summary_ops_v2.NoopSummaryWriter.__default_init__", "<builtin>.str", "tensorflow.python.framework.ops.device", "tensorflow.python.ops.summary_ops_v2.ResourceSummaryWriter.__init__", "tensorflow.python.framework.constant_op.constant", "functools.partial"], "tensorflow.python.ops.summary_ops_v2.always_record_summaries": ["tensorflow.python.ops.summary_ops_v2.record_if"], "tensorflow.python.ops.array_ops.placeholder_with_default": ["tensorflow.python.ops.gen_array_ops.placeholder_with_default"], "tensorflow.python.ops.summary_ops_v2.import_event": ["tensorflow.python.ops.gen_summary_ops.import_event", "tensorflow.python.eager.context.context"], "tensorflow.python.framework.constant_op.constant": ["tensorflow.python.framework.constant_op._constant_impl"], "tensorflow.python.util.compat.as_bytes": ["<builtin>.isinstance", "<builtin>.bytes", "<builtin>.TypeError"], "tensorflow.python.summary.writer.event_file_writer._EventLoggerThread.__init__": ["threading.Thread.__init__"], "tensorflow.python.summary.writer.writer.SummaryToEventTransformer.add_graph": ["<builtin>.ValueError", "<builtin>.isinstance", "tensorflow.python.summary.writer.writer.SummaryToEventTransformer._write_plugin_assets", "tensorflow.python.platform.tf_logging.warning", "<builtin>.TypeError", "tensorflow.python.summary.writer.writer.SummaryToEventTransformer._add_graph_def"], "tensorflow.python.summary.writer.writer.SummaryToEventTransformer.add_meta_graph": ["<builtin>.isinstance", "<builtin>.TypeError", "<builtin>.type", "tensorflow.python.summary.writer.writer.SummaryToEventTransformer._add_event"], "tensorflow.python.framework.meta_graph.create_meta_graph_def": ["<builtin>.TypeError", "<builtin>.isinstance", "<builtin>.type", "tensorflow.python.framework.ops.get_default_graph", "<builtin>.len", "tensorflow.python.framework.meta_graph.stripped_op_list_for_graph", "tensorflow.python.framework.meta_graph.strip_graph_default_valued_attrs", "tensorflow.python.framework.ops.get_from_proto_function", "tensorflow.python.framework.meta_graph.add_collection_def"], "tensorflow.python.ops.gen_array_ops.reshape_eager_fallback": ["tensorflow.python.eager.context.context", "tensorflow.python.eager.execute.args_to_matching_eager", "tensorflow.python.eager.execute.record_gradient"], "tensorflow.python.ops.gen_nn_ops.max_pool_eager_fallback": ["tensorflow.python.eager.context.context", "<builtin>.isinstance", "<builtin>.TypeError", "tensorflow.python.eager.execute.make_int", "tensorflow.python.eager.execute.make_str", "tensorflow.python.eager.execute.args_to_matching_eager", "tensorflow.python.eager.execute.record_gradient"], "tensorflow.python.ops.gen_math_ops.arg_max": ["tensorflow.python.eager.context.context", "six.raise_from", "tensorflow.python.ops.gen_math_ops.arg_max_eager_fallback", "tensorflow.python.eager.core._status_to_exception", "tensorflow.python.eager.execute.make_type", "tensorflow.python.eager.execute.record_gradient"], "tensorflow.python.framework.ops.internal_convert_to_tensor": ["<builtin>.isinstance", "tensorflow.python.eager.context.context", "tensorflow.python.framework.ops.get_default_graph", "<builtin>.RuntimeError", "tensorflow.python.framework.dtypes.as_dtype", "<builtin>.ValueError", "tensorflow.python.framework.tensor_conversion_registry.get", "<builtin>.type", "<builtin>.TypeError", "tensorflow.python.framework.ops._error_prefix"], "tensorflow.python.ops.gen_math_ops.cast_eager_fallback": ["tensorflow.python.eager.context.context", "tensorflow.python.eager.execute.make_type", "tensorflow.python.eager.execute.make_bool", "tensorflow.python.eager.execute.args_to_matching_eager", "tensorflow.python.eager.execute.record_gradient"], "tensorflow.python.ops.array_ops.zeros_like_impl": ["tensorflow.python.framework.ops.name_scope.__init__", "tensorflow.python.framework.ops.convert_to_tensor", "tensorflow.python.eager.context.executing_eagerly", "tensorflow.python.ops.array_ops.zeros", "tensorflow.python.ops.gen_array_ops.zeros_like", "tensorflow.python.framework.ops.device", "tensorflow.python.ops.array_ops.shape_internal"], "tensorflow.python.ops.gen_array_ops.where": ["tensorflow.python.eager.context.context", "six.raise_from", "tensorflow.python.ops.gen_array_ops.where_eager_fallback", "tensorflow.python.eager.core._status_to_exception", "tensorflow.python.eager.execute.record_gradient"], "tensorflow.python.ops.gen_math_ops.select": ["tensorflow.python.eager.context.context", "six.raise_from", "tensorflow.python.ops.gen_math_ops.select_eager_fallback", "tensorflow.python.eager.core._status_to_exception", "tensorflow.python.eager.execute.record_gradient"], "tensorflow.python.eager.execute.args_to_matching_eager": ["<builtin>.isinstance", "tensorflow.python.framework.ops._is_keras_symbolic_tensor", "tensorflow.python.eager.core._SymbolicException.__default_init__"], "tensorflow.python.ops.gen_math_ops.equal_eager_fallback": ["tensorflow.python.eager.context.context", "tensorflow.python.eager.execute.make_bool", "tensorflow.python.eager.execute.args_to_matching_eager", "tensorflow.python.eager.execute.record_gradient"], "tensorflow.python.ops.gen_nn_ops.bias_add_eager_fallback": ["tensorflow.python.eager.context.context", "tensorflow.python.eager.execute.make_str", "tensorflow.python.eager.execute.args_to_matching_eager", "tensorflow.python.eager.execute.record_gradient"], "tensorflow.python.ops.gen_math_ops.mean": ["tensorflow.python.eager.context.context", "six.raise_from", "tensorflow.python.ops.gen_math_ops.mean_eager_fallback", "tensorflow.python.eager.core._status_to_exception", "tensorflow.python.eager.execute.make_bool", "tensorflow.python.eager.execute.record_gradient"], "tensorflow.python.ops.gen_array_ops.placeholder_eager_fallback": ["tensorflow.python.eager.context.context", "tensorflow.python.eager.execute.make_type", "tensorflow.python.eager.execute.make_shape", "tensorflow.python.eager.execute.record_gradient"], "tensorflow.python.eager.execute.make_shape": ["tensorflow.python.framework.tensor_shape.as_shape", "<builtin>.TypeError", "<builtin>.ValueError"], "tensorflow.python.ops.control_flow_ops.group": ["tensorflow.python.eager.context.executing_eagerly", "<builtin>.ValueError", "tensorflow.python.framework.ops.name_scope.__init__", "tensorflow.python.util.nest.flatten", "<builtin>.sorted", "<builtin>.len", "tensorflow.python.ops.control_flow_ops._GroupControlDeps", "tensorflow.python.framework.ops.control_dependencies", "<builtin>.hasattr", "<builtin>.TypeError", "<builtin>.type"], "tensorflow.python.framework.ops.get_collection": ["tensorflow.python.framework.ops.get_default_graph"], "tensorflow.python.eager.monitoring.Counter.get_cell": ["tensorflow.python.eager.monitoring.CounterCell.__init__", "tensorflow.python.eager.monitoring.Metric.get_cell", "<builtin>.super"], "tensorflow.python.ops.gen_math_ops.conj": ["tensorflow.python.eager.context.context", "six.raise_from", "tensorflow.python.ops.gen_math_ops.conj_eager_fallback", "tensorflow.python.eager.core._status_to_exception", "tensorflow.python.eager.execute.record_gradient"], "tensorflow.python.ops.gen_math_ops.mat_mul_eager_fallback": ["tensorflow.python.eager.context.context", "tensorflow.python.eager.execute.make_bool", "tensorflow.python.eager.execute.args_to_matching_eager", "tensorflow.python.eager.execute.record_gradient"], "tensorflow.python.framework.ops.add_to_collection": ["tensorflow.python.framework.ops.get_default_graph"], "tensorflow.python.ops.variable_scope.get_variable_scope_store": ["tensorflow.python.framework.ops.get_collection", "tensorflow.python.ops.variable_scope._VariableScopeStore.__init__", "tensorflow.python.framework.ops.add_to_collection"], "tensorflow.python.framework.common_shapes.has_fully_defined_shape": ["<builtin>.isinstance"], "tensorflow.python.ops.gen_math_ops._min_eager_fallback": ["tensorflow.python.eager.context.context", "tensorflow.python.eager.execute.make_bool", "tensorflow.python.eager.execute.args_to_matching_eager", "tensorflow.python.eager.execute.record_gradient"], "tensorflow.python.framework.common_shapes.rank": ["<builtin>.isinstance"], "tensorflow.python.ops.math_ops.range": ["tensorflow.python.framework.ops.name_scope.__init__", "tensorflow.python.framework.ops.convert_to_tensor", "tensorflow.python.ops.gen_math_ops._range", "<builtin>.all", "<builtin>.max", "tensorflow.python.ops.math_ops.cast"], "tensorflow.python.ops.array_ops.rank": ["tensorflow.python.ops.array_ops.rank_internal"], "tensorflow.python.eager.context.Context.__init__": ["tensorflow.python.eager.context._AtomicCounter.increment_and_get", "tensorflow.python.eager.context._TensorCacheDeleter.__init__", "tensorflow.python.eager.context._TensorCaches.__init__", "tensorflow.python.eager.context._ThreadLocalData.__init__", "tensorflow.python.eager.context._ContextSwitchStack.__init__", "tensorflow.python.eager.context.Context.executing_eagerly", "threading.Lock", "<builtin>.ValueError", "tensorflow.python.eager.monitoring.Counter.get_cell"], "tensorflow.python.training.saver.BaseSaverBuilder._build_internal": ["<builtin>.ValueError", "tensorflow.python.eager.context.executing_eagerly", "tensorflow.python.training.saving.saveable_object_util.validate_and_slice_inputs", "tensorflow.python.framework.ops.name_scope.__init__", "tensorflow.python.ops.array_ops.placeholder_with_default", "tensorflow.python.training.saver.BaseSaverBuilder._GroupByDevices", "tensorflow.python.training.saver.BaseSaverBuilder._AddShardedSaveOps", "tensorflow.python.training.saver.BaseSaverBuilder._AddShardedRestoreOps", "tensorflow.python.training.saver.BaseSaverBuilder._AddSaveOps", "tensorflow.python.training.saver.BaseSaverBuilder._AddRestoreOps", "tensorflow.python.framework.ops.get_default_graph", "<builtin>.isinstance"], "tensorflow.python.training.saver.BulkSaverBuilder.__default_init__": ["tensorflow.python.training.saver.BaseSaverBuilder.__init__"], "tensorflow.python.ops.variables._all_saveable_objects": ["tensorflow.python.framework.ops.get_collection"], "tensorflow.python.framework.ops.prepend_name_scope": ["re.sub", "tensorflow.python.platform.tf_logging.warning"], "tensorflow.python.ops.array_ops.shape": ["tensorflow.python.ops.array_ops.shape_internal"], "tensorflow.python.framework.tensor_shape.as_shape": ["<builtin>.isinstance", "tensorflow.python.framework.tensor_shape.TensorShape.__init__"], "tensorflow.python.framework.tensor_shape.TensorShape.__init__": ["<builtin>.isinstance", "<builtin>.TypeError", "tensorflow.python.framework.tensor_shape.as_dimension", "<builtin>.iter"], "tensorflow.python.ops.gen_random_ops.random_uniform_int": ["tensorflow.python.eager.context.context", "six.raise_from", "tensorflow.python.ops.gen_random_ops.random_uniform_int_eager_fallback", "tensorflow.python.eager.core._status_to_exception", "tensorflow.python.eager.execute.make_int", "tensorflow.python.eager.execute.record_gradient"], "tensorflow.python.ops.gen_random_ops.random_uniform": ["tensorflow.python.eager.context.context", "six.raise_from", "tensorflow.python.ops.gen_random_ops.random_uniform_eager_fallback", "tensorflow.python.eager.core._status_to_exception", "tensorflow.python.eager.execute.make_type", "tensorflow.python.eager.execute.make_int", "tensorflow.python.eager.execute.record_gradient"], "tensorflow.python.platform.tf_logging._GetFileAndLine": ["tensorflow.python.platform.tf_logging._get_caller"], "tensorflow.python.platform.tf_logging.log_if": ["tensorflow.python.platform.tf_logging.vlog"], "tensorflow.python.framework.tensor_util._ConstantValue": ["<builtin>.isinstance", "<builtin>.TypeError", "<builtin>.type", "tensorflow.python.framework.tensor_util.MakeNdarray", "numpy.array", "numpy.prod", "numpy.ndarray", "tensorflow.python.framework.tensor_util.constant_value", "numpy.arange", "tensorflow.python.framework.dtypes.as_dtype", "numpy.concatenate", "numpy.full", "numpy.equal", "numpy.not_equal"], "tensorflow.python.ops.gen_random_ops.truncated_normal": ["tensorflow.python.eager.context.context", "six.raise_from", "tensorflow.python.ops.gen_random_ops.truncated_normal_eager_fallback", "tensorflow.python.eager.core._status_to_exception", "tensorflow.python.eager.execute.make_type", "tensorflow.python.eager.execute.make_int", "tensorflow.python.eager.execute.record_gradient"], "tensorflow.python.framework.ops._DefaultGraphStack.get_default": ["tensorflow.python.framework.ops._DefaultStack.get_default", "<builtin>.super", "tensorflow.python.framework.ops._DefaultGraphStack._GetGlobalDefaultGraph"], "tensorflow.python.framework.ops.is_dense_tensor_like": ["<builtin>.isinstance"], "tensorflow.python.framework.tensor_shape.unknown_shape": ["<builtin>.TypeError", "tensorflow.python.framework.tensor_shape.TensorShape.__init__", "tensorflow.python.framework.tensor_shape.Dimension.__init__"], "tensorflow.python.framework.tensor_shape.TensorShape.concatenate": ["tensorflow.python.framework.tensor_shape.as_shape", "tensorflow.python.framework.tensor_shape.unknown_shape", "tensorflow.python.framework.tensor_shape.TensorShape.__init__"], "tensorflow.python.framework.tensor_shape.Dimension.__init__": ["<builtin>.isinstance", "<builtin>.TypeError", "<builtin>.int", "<builtin>.ValueError"], "tensorflow.python.framework.tensor_shape.TensorShape.merge_with": ["tensorflow.python.framework.tensor_shape.as_shape", "tensorflow.python.framework.tensor_shape.TensorShape.assert_same_rank", "<builtin>.enumerate", "tensorflow.python.framework.tensor_shape.TensorShape.__init__", "<builtin>.ValueError"], "tensorflow.python.framework.ops.device": ["tensorflow.python.eager.context.executing_eagerly", "<builtin>.callable", "tensorflow.python.eager.context.device", "tensorflow.python.framework.ops.executing_eagerly_outside_functions", "<builtin>.RuntimeError", "tensorflow.python.framework.ops.get_default_graph"], "tensorflow.python.ops.summary_ops_v2.ResourceSummaryWriter.__init__": ["tensorflow.python.ops.gen_summary_ops.summary_writer", "tensorflow.python.eager.context.executing_eagerly", "tensorflow.python.ops.resource_variable_ops.EagerResourceDeleter.__init__", "tensorflow.python.framework.ops.add_to_collection"], "tensorflow.python.ops.summary_ops_v2.record_if": ["tensorflow.python.eager.context.context"], "tensorflow.python.ops.gen_array_ops.placeholder_with_default": ["tensorflow.python.eager.context.context", "six.raise_from", "tensorflow.python.ops.gen_array_ops.placeholder_with_default_eager_fallback", "tensorflow.python.eager.core._status_to_exception", "tensorflow.python.eager.execute.make_shape", "tensorflow.python.eager.execute.record_gradient"], "tensorflow.python.ops.gen_summary_ops.import_event": ["tensorflow.python.eager.context.context", "six.raise_from", "tensorflow.python.ops.gen_summary_ops.import_event_eager_fallback", "tensorflow.python.eager.core._status_to_exception"], "tensorflow.python.framework.constant_op._constant_impl": ["tensorflow.python.eager.context.context", "tensorflow.python.framework.constant_op.convert_to_eager_tensor", "tensorflow.python.framework.tensor_shape.as_shape", "<builtin>.TypeError", "tensorflow.python.framework.constant_op._eager_reshape", "tensorflow.python.framework.constant_op._eager_identity", "tensorflow.python.framework.constant_op._eager_fill", "tensorflow.python.framework.ops.device", "<builtin>.tuple", "tensorflow.python.framework.ops.get_default_graph", "tensorflow.python.framework.tensor_util.make_tensor_proto"], "tensorflow.python.summary.writer.writer.SummaryToEventTransformer._write_plugin_assets": ["tensorflow.python.summary.plugin_asset.get_all_plugin_assets", "os.path.join"], "tensorflow.python.summary.writer.writer.SummaryToEventTransformer._add_graph_def": ["tensorflow.python.summary.writer.writer.SummaryToEventTransformer._add_event"], "tensorflow.python.summary.writer.writer.SummaryToEventTransformer._add_event": ["time.time", "<builtin>.int"], "tensorflow.python.framework.meta_graph.stripped_op_list_for_graph": ["tensorflow.python.framework.meta_graph.ops_used_by_graph_def", "tensorflow.python.framework.op_def_registry.get_registered_ops", "<builtin>.ValueError", "<builtin>.sorted"], "tensorflow.python.framework.meta_graph.strip_graph_default_valued_attrs": ["tensorflow.python.framework.op_def_registry.get_registered_ops", "<builtin>.set", "tensorflow.python.framework.meta_graph._is_default_attr_value"], "tensorflow.python.framework.ops.get_from_proto_function": ["tensorflow.python.framework.registry.Registry.lookup"], "tensorflow.python.framework.meta_graph.add_collection_def": ["<builtin>.TypeError", "<builtin>.isinstance", "<builtin>.type", "tensorflow.python.platform.tf_logging.warning", "tensorflow.python.framework.ops.get_default_graph", "tensorflow.python.framework.meta_graph._should_include_node", "tensorflow.python.framework.ops.get_to_proto_function", "tensorflow.python.framework.ops.get_collection_proto_type", "tensorflow.python.framework.meta_graph._get_kind_name", "<builtin>.str", "tensorflow.python.framework.ops.strip_name_scope", "tensorflow.python.util.compat.as_bytes", "<builtin>.getattr"], "tensorflow.python.ops.gen_math_ops.arg_max_eager_fallback": ["tensorflow.python.eager.context.context", "tensorflow.python.eager.execute.make_type", "tensorflow.python.eager.execute.args_to_matching_eager", "tensorflow.python.eager.execute.record_gradient"], "tensorflow.python.framework.tensor_conversion_registry.get": ["<builtin>.issubclass", "<builtin>.sorted"], "tensorflow.python.ops.array_ops.zeros": ["tensorflow.python.framework.dtypes.as_dtype", "tensorflow.python.framework.ops.name_scope.__init__", "tensorflow.python.ops.array_ops.fill", "<builtin>.isinstance", "tensorflow.python.ops.array_ops.reshape", "tensorflow.python.framework.constant_op.constant", "tensorflow.python.ops.array_ops._constant_if_small", "tensorflow.python.framework.constant_op._tensor_shape_tensor_conversion_function", "tensorflow.python.framework.tensor_shape.TensorShape.__init__", "tensorflow.python.framework.ops.convert_to_tensor"], "tensorflow.python.ops.gen_array_ops.zeros_like": ["tensorflow.python.eager.context.context", "six.raise_from", "tensorflow.python.ops.gen_array_ops.zeros_like_eager_fallback", "tensorflow.python.eager.core._status_to_exception", "tensorflow.python.eager.execute.record_gradient"], "tensorflow.python.ops.array_ops.shape_internal": ["tensorflow.python.framework.ops.name_scope.__init__", "<builtin>.isinstance", "tensorflow.python.ops.gen_math_ops.cast", "tensorflow.python.ops.gen_array_ops.shape", "tensorflow.python.eager.context.executing_eagerly", "tensorflow.python.framework.ops.convert_to_tensor", "tensorflow.python.framework.constant_op.constant"], "tensorflow.python.ops.gen_array_ops.where_eager_fallback": ["tensorflow.python.eager.context.context", "tensorflow.python.eager.execute.args_to_matching_eager", "tensorflow.python.eager.execute.record_gradient"], "tensorflow.python.ops.gen_math_ops.select_eager_fallback": ["tensorflow.python.eager.context.context", "tensorflow.python.eager.execute.args_to_matching_eager", "tensorflow.python.framework.ops.convert_to_tensor", "tensorflow.python.eager.execute.record_gradient"], "tensorflow.python.framework.ops._is_keras_symbolic_tensor": ["<builtin>.hasattr", "<builtin>.getattr"], "tensorflow.python.ops.gen_math_ops.mean_eager_fallback": ["tensorflow.python.eager.context.context", "tensorflow.python.eager.execute.make_bool", "tensorflow.python.eager.execute.args_to_matching_eager", "tensorflow.python.eager.execute.record_gradient"], "tensorflow.python.ops.control_flow_ops._GroupControlDeps": ["tensorflow.python.framework.ops.control_dependencies", "tensorflow.python.framework.ops.device"], "tensorflow.python.framework.ops.control_dependencies": ["tensorflow.python.eager.context.executing_eagerly", "tensorflow.python.framework.ops.NullContextmanager.__init__", "<builtin>.callable", "tensorflow.python.framework.ops.get_default_graph"], "tensorflow.python.eager.monitoring.Metric.get_cell": ["<builtin>.len", "<builtin>.ValueError"], "tensorflow.python.ops.gen_math_ops.conj_eager_fallback": ["tensorflow.python.eager.context.context", "tensorflow.python.eager.execute.args_to_matching_eager", "tensorflow.python.eager.execute.record_gradient"], "tensorflow.python.ops.variable_scope._VariableScopeStore.__init__": ["<builtin>.super", "tensorflow.python.ops.variable_scope.VariableScope.__init__"], "tensorflow.python.ops.gen_math_ops._range": ["tensorflow.python.eager.context.context", "six.raise_from", "tensorflow.python.ops.gen_math_ops._range_eager_fallback", "tensorflow.python.eager.core._status_to_exception", "tensorflow.python.eager.execute.record_gradient"], "tensorflow.python.ops.array_ops.rank_internal": ["tensorflow.python.framework.ops.name_scope.__init__", "<builtin>.isinstance", "tensorflow.python.ops.gen_array_ops.size", "tensorflow.python.framework.ops.convert_to_tensor", "tensorflow.python.ops.gen_array_ops.rank", "tensorflow.python.framework.constant_op.constant"], "tensorflow.python.eager.context._TensorCaches.__init__": ["<builtin>.super"], "tensorflow.python.eager.context._ThreadLocalData.__init__": ["<builtin>.super"], "tensorflow.python.eager.context._ContextSwitchStack.__init__": ["<builtin>.super", "tensorflow.python.eager.context._ContextSwitchStack.push"], "tensorflow.python.training.saving.saveable_object_util.validate_and_slice_inputs": ["<builtin>.isinstance", "tensorflow.python.training.saving.saveable_object_util.op_list_to_dict", "tensorflow.python.util.object_identity.ObjectIdentitySet.__init__", "<builtin>.sorted", "tensorflow.python.training.saving.saveable_object_util.saveable_objects_for_op", "tensorflow.python.training.saving.saveable_object_util._add_saveable"], "tensorflow.python.training.saver.BaseSaverBuilder._GroupByDevices": ["collections.defaultdict", "<builtin>.set", "<builtin>.len", "<builtin>.ValueError", "tensorflow.python.framework.device.canonical_name", "<builtin>.sorted"], "tensorflow.python.training.saver.BaseSaverBuilder._AddShardedSaveOps": ["tensorflow.python.training.saver.BaseSaverBuilder._AddShardedSaveOpsForV2", "<builtin>.len", "tensorflow.python.framework.constant_op.constant", "<builtin>.enumerate", "tensorflow.python.framework.ops.device", "tensorflow.python.training.saver.BaseSaverBuilder.sharded_filename", "tensorflow.python.training.saver.BaseSaverBuilder._AddSaveOps", "tensorflow.python.framework.ops.control_dependencies", "tensorflow.python.ops.gen_io_ops.sharded_filespec"], "tensorflow.python.training.saver.BaseSaverBuilder._AddShardedRestoreOps": ["<builtin>.enumerate", "tensorflow.python.framework.ops.device", "tensorflow.python.training.saver.BaseSaverBuilder._AddRestoreOps", "tensorflow.python.ops.control_flow_ops.group"], "tensorflow.python.training.saver.BaseSaverBuilder._AddSaveOps": ["tensorflow.python.training.saver.BaseSaverBuilder.save_op", "tensorflow.python.ops.control_flow_ops.with_dependencies"], "tensorflow.python.training.saver.BaseSaverBuilder._AddRestoreOps": ["tensorflow.python.training.saver.BaseSaverBuilder.bulk_restore", "<builtin>.len", "tensorflow.python.ops.array_ops.shape", "tensorflow.python.ops.control_flow_ops.group"], "tensorflow.python.framework.tensor_shape.as_dimension": ["<builtin>.isinstance", "tensorflow.python.framework.tensor_shape.Dimension.__init__"], "tensorflow.python.ops.gen_random_ops.random_uniform_int_eager_fallback": ["tensorflow.python.eager.context.context", "tensorflow.python.eager.execute.make_int", "tensorflow.python.eager.execute.args_to_matching_eager", "tensorflow.python.eager.execute.record_gradient"], "tensorflow.python.ops.gen_random_ops.random_uniform_eager_fallback": ["tensorflow.python.eager.context.context", "tensorflow.python.eager.execute.make_type", "tensorflow.python.eager.execute.make_int", "tensorflow.python.eager.execute.args_to_matching_eager", "tensorflow.python.eager.execute.record_gradient"], "tensorflow.python.platform.tf_logging._get_caller": ["sys._getframe"], "tensorflow.python.platform.tf_logging.vlog": ["tensorflow.python.platform.tf_logging.get_logger"], "tensorflow.python.framework.tensor_util.MakeNdarray": ["numpy.prod", "tensorflow.python.framework.dtypes.as_dtype", "numpy.frombuffer", "<builtin>.list", "<builtin>.len", "numpy.array", "numpy.fromiter", "<builtin>.iter", "<builtin>.complex", "<builtin>.TypeError", "<builtin>.zip", "numpy.zeros", "numpy.pad"], "tensorflow.python.ops.gen_random_ops.truncated_normal_eager_fallback": ["tensorflow.python.eager.context.context", "tensorflow.python.eager.execute.make_type", "tensorflow.python.eager.execute.make_int", "tensorflow.python.eager.execute.args_to_matching_eager", "tensorflow.python.eager.execute.record_gradient"], "tensorflow.python.framework.ops._DefaultStack.get_default": ["<builtin>.len"], "tensorflow.python.framework.ops._DefaultGraphStack._GetGlobalDefaultGraph": ["tensorflow.python.framework.ops.Graph.__init__"], "tensorflow.python.framework.tensor_shape.TensorShape.assert_same_rank": ["tensorflow.python.framework.tensor_shape.as_shape", "<builtin>.ValueError"], "tensorflow.python.eager.context.device": ["tensorflow.python.eager.context.ensure_initialized", "tensorflow.python.eager.context.context"], "tensorflow.python.framework.ops.executing_eagerly_outside_functions": ["tensorflow.python.eager.context.executing_eagerly", "tensorflow.python.framework.ops._get_outer_context_and_inner_device_stack"], "tensorflow.python.ops.gen_summary_ops.summary_writer": ["tensorflow.python.eager.context.context", "six.raise_from", "tensorflow.python.ops.gen_summary_ops.summary_writer_eager_fallback", "tensorflow.python.eager.core._status_to_exception", "tensorflow.python.eager.execute.make_str", "tensorflow.python.eager.execute.record_gradient"], "tensorflow.python.ops.resource_variable_ops.EagerResourceDeleter.__init__": ["<builtin>.isinstance", "<builtin>.ValueError", "tensorflow.python.eager.context.context"], "tensorflow.python.ops.gen_array_ops.placeholder_with_default_eager_fallback": ["tensorflow.python.eager.context.context", "tensorflow.python.eager.execute.make_shape", "tensorflow.python.eager.execute.args_to_matching_eager", "tensorflow.python.eager.execute.record_gradient"], "tensorflow.python.ops.gen_summary_ops.import_event_eager_fallback": ["tensorflow.python.eager.context.context", "tensorflow.python.framework.ops.convert_to_tensor"], "tensorflow.python.framework.constant_op.convert_to_eager_tensor": ["<builtin>.isinstance", "<builtin>.TypeError", "tensorflow.python.framework.dtypes.as_dtype"], "tensorflow.python.framework.constant_op._eager_reshape": ["tensorflow.python.eager.execute.args_to_matching_eager"], "tensorflow.python.framework.constant_op._eager_fill": ["tensorflow.python.framework.constant_op.convert_to_eager_tensor"], "tensorflow.python.framework.tensor_util.make_tensor_proto": ["<builtin>.ValueError", "<builtin>.isinstance", "tensorflow.python.framework.dtypes.as_dtype", "tensorflow.python.framework.tensor_util._is_array_like", "numpy.asarray", "numpy.empty", "tensorflow.python.framework.tensor_util._AssertCompatible", "numpy.array", "numpy.prod", "numpy.array_equal", "<builtin>.list", "tensorflow.python.framework.tensor_util._GetDenseDimensions", "<builtin>.TypeError", "<builtin>.hasattr", "<builtin>.int", "<builtin>.tuple", "tensorflow.python.framework.tensor_shape.as_shape", "tensorflow.python.framework.tensor_util._FlattenToStrings", "tensorflow.python.util.compat.as_bytes", "<builtin>.type", "tensorflow.python.framework.tensor_util.GetNumpyAppendFn"], "tensorflow.python.summary.plugin_asset.get_all_plugin_assets": ["tensorflow.python.framework.ops.get_default_graph", "<builtin>.len", "<builtin>.ValueError"], "tensorflow.python.framework.meta_graph.ops_used_by_graph_def": ["<builtin>.set"], "tensorflow.python.framework.registry.Registry.lookup": ["<builtin>.LookupError"], "tensorflow.python.framework.meta_graph._should_include_node": ["<builtin>.isinstance"], "tensorflow.python.framework.ops.get_to_proto_function": ["tensorflow.python.framework.registry.Registry.lookup"], "tensorflow.python.framework.ops.get_collection_proto_type": ["tensorflow.python.framework.registry.Registry.lookup"], "tensorflow.python.framework.meta_graph._get_kind_name": ["<builtin>.isinstance"], "tensorflow.python.framework.ops.strip_name_scope": ["re.sub", "tensorflow.python.platform.tf_logging.warning"], "tensorflow.python.ops.array_ops.fill": ["tensorflow.python.ops.gen_array_ops.fill", "tensorflow.python.framework.tensor_util.maybe_set_static_shape"], "tensorflow.python.ops.array_ops._constant_if_small": ["numpy.prod", "tensorflow.python.framework.constant_op.constant"], "tensorflow.python.framework.constant_op._tensor_shape_tensor_conversion_function": ["<builtin>.ValueError", "<builtin>.TypeError", "tensorflow.python.framework.constant_op.constant"], "tensorflow.python.ops.gen_array_ops.zeros_like_eager_fallback": ["tensorflow.python.eager.context.context", "tensorflow.python.eager.execute.args_to_matching_eager", "tensorflow.python.eager.execute.record_gradient"], "tensorflow.python.ops.gen_array_ops.shape": ["tensorflow.python.eager.context.context", "six.raise_from", "tensorflow.python.ops.gen_array_ops.shape_eager_fallback", "tensorflow.python.eager.core._status_to_exception", "tensorflow.python.eager.execute.make_type", "tensorflow.python.eager.execute.record_gradient"], "tensorflow.python.ops.variable_scope.VariableScope.__init__": ["tensorflow.python.eager.context.executing_eagerly", "<builtin>.NotImplementedError"], "tensorflow.python.ops.gen_math_ops._range_eager_fallback": ["tensorflow.python.eager.context.context", "tensorflow.python.eager.execute.args_to_matching_eager", "tensorflow.python.eager.execute.record_gradient"], "tensorflow.python.ops.gen_array_ops.size": ["tensorflow.python.eager.context.context", "six.raise_from", "tensorflow.python.ops.gen_array_ops.size_eager_fallback", "tensorflow.python.eager.core._status_to_exception", "tensorflow.python.eager.execute.make_type", "tensorflow.python.eager.execute.record_gradient"], "tensorflow.python.ops.gen_array_ops.rank": ["tensorflow.python.eager.context.context", "six.raise_from", "tensorflow.python.ops.gen_array_ops.rank_eager_fallback", "tensorflow.python.eager.core._status_to_exception", "tensorflow.python.eager.execute.record_gradient"], "tensorflow.python.training.saving.saveable_object_util.op_list_to_dict": ["<builtin>.isinstance", "<builtin>.TypeError", "<builtin>.sorted", "<builtin>.ValueError", "tensorflow.python.training.saving.saveable_object_util.op_list_to_dict", "<builtin>.getattr", "<builtin>.callable", "tensorflow.python.framework.ops.internal_convert_to_tensor", "tensorflow.python.training.saving.saveable_object_util._tensor_comes_from_variable", "<builtin>.type"], "tensorflow.python.training.saving.saveable_object_util.saveable_objects_for_op": ["<builtin>.isinstance", "<builtin>.TypeError", "<builtin>.list", "<builtin>.ValueError", "tensorflow.python.training.saving.saveable_object_util.saveable_objects_for_op", "tensorflow.python.eager.context.executing_eagerly", "tensorflow.python.framework.ops.internal_convert_to_tensor", "tensorflow.python.training.saving.saveable_object_util.ReferenceVariableSaveable.__init__", "tensorflow.python.training.saving.saveable_object_util.ResourceVariableSaveable.__init__", "<builtin>.callable", "tensorflow.python.training.saving.saveable_object_util._tensor_comes_from_variable", "<builtin>.type"], "tensorflow.python.training.saving.saveable_object_util._add_saveable": ["<builtin>.ValueError"], "tensorflow.python.framework.device.canonical_name": ["tensorflow.python.framework.device.is_device_spec"], "tensorflow.python.training.saver.BaseSaverBuilder._AddShardedSaveOpsForV2": ["uuid.uuid4", "<builtin>.len", "tensorflow.python.framework.constant_op.constant", "<builtin>.enumerate", "tensorflow.python.framework.ops.device", "tensorflow.python.training.saver.BaseSaverBuilder.sharded_filename", "tensorflow.python.training.saving.saveable_object_util.set_cpu0", "tensorflow.python.training.saver.BaseSaverBuilder._AddSaveOps", "tensorflow.python.framework.ops.control_dependencies", "tensorflow.python.ops.gen_io_ops.merge_v2_checkpoints", "tensorflow.python.ops.array_ops.identity"], "tensorflow.python.training.saver.BaseSaverBuilder.sharded_filename": ["tensorflow.python.ops.gen_io_ops.sharded_filename"], "tensorflow.python.ops.gen_io_ops.sharded_filespec": ["tensorflow.python.eager.context.context", "six.raise_from", "tensorflow.python.ops.gen_io_ops.sharded_filespec_eager_fallback", "tensorflow.python.eager.core._status_to_exception", "tensorflow.python.eager.execute.record_gradient"], "tensorflow.python.training.saver.BaseSaverBuilder.save_op": ["tensorflow.python.ops.io_ops._save", "<builtin>.RuntimeError"], "tensorflow.python.ops.control_flow_ops.with_dependencies": ["tensorflow.python.eager.context.executing_eagerly", "tensorflow.python.framework.ops.name_scope.__init__", "tensorflow.python.framework.ops.colocate_with", "<builtin>.list", "tensorflow.python.framework.ops.control_dependencies", "tensorflow.python.framework.ops.convert_to_tensor_or_composite", "<builtin>.isinstance", "tensorflow.python.ops.control_flow_ops._Identity"], "tensorflow.python.training.saver.BaseSaverBuilder.bulk_restore": ["tensorflow.python.training.saving.saveable_object_util.set_cpu0", "tensorflow.python.framework.ops.device", "tensorflow.python.training.saver.BaseSaverBuilder.restore_op"], "tensorflow.python.eager.context.ensure_initialized": ["tensorflow.python.eager.context.context"], "tensorflow.python.framework.ops._get_outer_context_and_inner_device_stack": ["tensorflow.python.framework.ops.get_default_graph", "<builtin>.reversed", "<builtin>.RuntimeError", "tensorflow.python.framework.ops._DefaultGraphStack._GetGlobalDefaultGraph", "tensorflow.python.eager.context.context"], "tensorflow.python.ops.gen_summary_ops.summary_writer_eager_fallback": ["tensorflow.python.eager.context.context", "tensorflow.python.eager.execute.make_str", "tensorflow.python.eager.execute.record_gradient"], "tensorflow.python.framework.tensor_util._is_array_like": ["<builtin>.isinstance", "<builtin>.callable", "<builtin>.getattr", "<builtin>.memoryview"], "tensorflow.python.framework.tensor_util._AssertCompatible": ["<builtin>.TypeError", "<builtin>.repr", "<builtin>.type"], "tensorflow.python.framework.tensor_util._GetDenseDimensions": ["<builtin>.isinstance", "tensorflow.python.framework.tensor_util._GetDenseDimensions", "<builtin>.len"], "tensorflow.python.framework.tensor_util._FlattenToStrings": ["<builtin>.isinstance", "tensorflow.python.framework.tensor_util._FlattenToStrings"], "tensorflow.python.framework.tensor_util.GetNumpyAppendFn": ["tensorflow.python.framework.tensor_util.GetFromNumpyDTypeDict"], "tensorflow.python.ops.gen_array_ops.fill": ["tensorflow.python.eager.context.context", "six.raise_from", "tensorflow.python.ops.gen_array_ops.fill_eager_fallback", "tensorflow.python.eager.core._status_to_exception", "tensorflow.python.eager.execute.record_gradient"], "tensorflow.python.ops.gen_array_ops.shape_eager_fallback": ["tensorflow.python.eager.context.context", "tensorflow.python.eager.execute.make_type", "tensorflow.python.eager.execute.args_to_matching_eager", "tensorflow.python.eager.execute.record_gradient"], "tensorflow.python.ops.gen_array_ops.size_eager_fallback": ["tensorflow.python.eager.context.context", "tensorflow.python.eager.execute.make_type", "tensorflow.python.eager.execute.args_to_matching_eager", "tensorflow.python.eager.execute.record_gradient"], "tensorflow.python.ops.gen_array_ops.rank_eager_fallback": ["tensorflow.python.eager.context.context", "tensorflow.python.eager.execute.args_to_matching_eager", "tensorflow.python.eager.execute.record_gradient"], "tensorflow.python.training.saving.saveable_object_util._tensor_comes_from_variable": ["<builtin>.isinstance"], "tensorflow.python.training.saving.saveable_object_util.ReferenceVariableSaveable.__init__": ["tensorflow.python.training.saving.saveable_object.SaveSpec.__init__", "<builtin>.super"], "tensorflow.python.training.saving.saveable_object_util.ResourceVariableSaveable.__init__": ["<builtin>.isinstance", "tensorflow.python.ops.resource_variable_ops.is_resource_variable", "<builtin>.ValueError", "<builtin>.repr", "tensorflow.python.framework.ops.device", "tensorflow.python.ops.array_ops.identity", "tensorflow.python.training.saving.saveable_object.SaveSpec.__init__", "<builtin>.super"], "tensorflow.python.framework.device.is_device_spec": ["<builtin>.isinstance"], "tensorflow.python.ops.gen_io_ops.merge_v2_checkpoints": ["tensorflow.python.eager.context.context", "six.raise_from", "tensorflow.python.ops.gen_io_ops.merge_v2_checkpoints_eager_fallback", "tensorflow.python.eager.core._status_to_exception", "tensorflow.python.eager.execute.make_bool"], "tensorflow.python.ops.array_ops.identity": ["tensorflow.python.eager.context.executing_eagerly", "tensorflow.python.framework.ops.convert_to_tensor", "<builtin>.hasattr", "tensorflow.python.ops.gen_array_ops.identity"], "tensorflow.python.ops.gen_io_ops.sharded_filename": ["tensorflow.python.eager.context.context", "six.raise_from", "tensorflow.python.ops.gen_io_ops.sharded_filename_eager_fallback", "tensorflow.python.eager.core._status_to_exception", "tensorflow.python.eager.execute.record_gradient"], "tensorflow.python.ops.gen_io_ops.sharded_filespec_eager_fallback": ["tensorflow.python.eager.context.context", "tensorflow.python.framework.ops.convert_to_tensor", "tensorflow.python.eager.execute.record_gradient"], "tensorflow.python.ops.io_ops._save": ["tensorflow.python.ops.gen_io_ops.save", "tensorflow.python.ops.gen_io_ops.save_slices"], "tensorflow.python.framework.ops.colocate_with": ["tensorflow.python.framework.ops._colocate_with_for_gradient"], "tensorflow.python.framework.ops.convert_to_tensor_or_composite": ["tensorflow.python.framework.ops.internal_convert_to_tensor_or_composite"], "tensorflow.python.ops.control_flow_ops._Identity": ["tensorflow.python.framework.ops.internal_convert_to_tensor_or_composite", "<builtin>.isinstance", "tensorflow.python.ops.gen_array_ops.ref_identity", "tensorflow.python.ops.array_ops.identity", "tensorflow.python.util.nest.map_structure", "<builtin>.TypeError", "<builtin>.type"], "tensorflow.python.framework.tensor_util.GetFromNumpyDTypeDict": ["six.iteritems"], "tensorflow.python.ops.gen_array_ops.fill_eager_fallback": ["tensorflow.python.eager.context.context", "tensorflow.python.eager.execute.args_to_matching_eager", "tensorflow.python.eager.execute.record_gradient"], "tensorflow.python.training.saving.saveable_object.SaveSpec.__init__": ["<builtin>.callable", "<builtin>.AssertionError"], "tensorflow.python.ops.resource_variable_ops.is_resource_variable": ["<builtin>.isinstance", "<builtin>.hasattr"], "tensorflow.python.ops.gen_io_ops.merge_v2_checkpoints_eager_fallback": ["tensorflow.python.eager.context.context", "tensorflow.python.eager.execute.make_bool", "tensorflow.python.framework.ops.convert_to_tensor"], "tensorflow.python.ops.gen_array_ops.identity": ["tensorflow.python.eager.context.context", "six.raise_from", "tensorflow.python.ops.gen_array_ops.identity_eager_fallback", "tensorflow.python.eager.core._status_to_exception", "tensorflow.python.eager.execute.record_gradient"], "tensorflow.python.ops.gen_io_ops.sharded_filename_eager_fallback": ["tensorflow.python.eager.context.context", "tensorflow.python.framework.ops.convert_to_tensor", "tensorflow.python.eager.execute.record_gradient"], "tensorflow.python.ops.gen_io_ops.save": ["tensorflow.python.eager.context.context", "six.raise_from", "tensorflow.python.ops.gen_io_ops.save_eager_fallback", "tensorflow.python.eager.core._status_to_exception"], "tensorflow.python.ops.gen_io_ops.save_slices": ["tensorflow.python.eager.context.context", "six.raise_from", "tensorflow.python.ops.gen_io_ops.save_slices_eager_fallback", "tensorflow.python.eager.core._status_to_exception"], "tensorflow.python.framework.ops._colocate_with_for_gradient": ["tensorflow.python.eager.context.executing_eagerly", "tensorflow.python.framework.ops.get_default_graph", "<builtin>.isinstance", "tensorflow.python.framework.ops.device", "tensorflow.python.framework.ops.NullContextmanager.__init__", "<builtin>.hasattr", "<builtin>.ValueError"], "tensorflow.python.framework.ops.internal_convert_to_tensor_or_composite": ["<builtin>.isinstance", "<builtin>.getattr", "tensorflow.python.framework.ops.internal_convert_to_tensor", "<builtin>.ValueError", "tensorflow.python.framework.dtypes.as_dtype", "<builtin>.str"], "tensorflow.python.ops.gen_array_ops.ref_identity": ["tensorflow.python.eager.context.context", "<builtin>.RuntimeError", "tensorflow.python.eager.execute.record_gradient"], "tensorflow.python.util.nest.map_structure": ["<builtin>.callable", "<builtin>.TypeError", "<builtin>.ValueError", "tensorflow.python.util.nest.assert_same_structure", "tensorflow.python.util.nest.flatten", "<builtin>.zip", "tensorflow.python.util.nest.pack_sequence_as"], "tensorflow.python.ops.gen_array_ops.identity_eager_fallback": ["tensorflow.python.eager.context.context", "tensorflow.python.eager.execute.args_to_matching_eager", "tensorflow.python.eager.execute.record_gradient"], "tensorflow.python.ops.gen_io_ops.save_eager_fallback": ["tensorflow.python.eager.context.context", "tensorflow.python.eager.execute.convert_to_mixed_eager_tensors", "tensorflow.python.framework.ops.convert_to_tensor", "<builtin>.list"], "tensorflow.python.ops.gen_io_ops.save_slices_eager_fallback": ["tensorflow.python.eager.context.context", "tensorflow.python.eager.execute.convert_to_mixed_eager_tensors", "tensorflow.python.framework.ops.convert_to_tensor", "<builtin>.list"], "tensorflow.python.util.nest.assert_same_structure": ["<builtin>.str", "tensorflow.python.util.nest.map_structure", "<builtin>.type"], "tensorflow.python.util.nest.pack_sequence_as": ["<builtin>.TypeError", "<builtin>.len", "<builtin>.ValueError", "tensorflow.python.util.nest._packed_nest_with_indices", "tensorflow.python.util.nest.flatten", "tensorflow.python.util.nest._sequence_like"], "tensorflow.python.eager.execute.convert_to_mixed_eager_tensors": ["tensorflow.python.framework.ops.internal_convert_to_tensor"], "tensorflow.python.util.nest._packed_nest_with_indices": ["tensorflow.python.util.nest._yield_value", "tensorflow.python.util.nest._packed_nest_with_indices", "tensorflow.python.util.nest._sequence_like"], "tensorflow.python.util.nest._sequence_like": ["<builtin>.dict", "<builtin>.type", "<builtin>.zip", "collections.defaultdict", "tensorflow.python.util.nest._sorted", "<builtin>.list", "tensorflow.python.util.nest._is_namedtuple", "<builtin>.len", "<builtin>.isinstance", "tensorflow.python.util.nest._sequence_like"], "tensorflow.python.util.nest._yield_value": ["tensorflow.python.util.nest._yield_sorted_items"], "tensorflow.python.util.nest._sorted": ["<builtin>.sorted", "<builtin>.TypeError"], "tensorflow.python.util.nest._yield_sorted_items": ["<builtin>.isinstance", "tensorflow.python.util.nest._sorted", "tensorflow.python.util.nest._get_attrs_items", "tensorflow.python.util.nest._is_namedtuple", "<builtin>.enumerate", "<builtin>.getattr", "<builtin>.type"], "tensorflow.python.util.nest._get_attrs_items": ["<builtin>.getattr"]}